rmw_fastrtps_cpp/src/rmw_logging.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright 2016 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "rmw/rmw.h"
+
+#include "fastrtps/log/Log.h"
+
+extern "C"
+{
+using eprosima::fastrtps::Log;
+
+eprosima::fastrtps::Log::Kind rmw_convert_severity_type(rmw_log_severity_t severity)
```

Comment:
As mentioned before: functions not defined in the rmw interface should not be prefixed with `rmw_`.